tests/jsx_escape/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -42,9 +42,9 @@ raw_amp = <span>foo & bar</span>
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 many_nbsp = <div>&nbsp; &nbsp; </div>;
 single_nbsp = <div>&nbsp;</div>;
-many_raw_nbsp = <div>&nbsp;&nbsp;&nbsp;</div>;
+many_raw_nbsp = <div>   </div>;
```

Comment:
The diff you are highlighting was coming from the previous implementation. If you take a look at the snapshot, you'll see that that there are no `&nbsp;` in this case but only spaces (maybe should have been called `many_raw_spaces`): https://github.com/yamafaktory/prettier/blob/be94689bb0a0f18b34faddc3effb205f57745fa6/tests/jsx_escape/__snapshots__/jsfmt.spec.js.snap#L37.